internal/services/netapp/netapp_volume_resource.go

```diff
@@ -262,7 +263,7 @@ func resourceNetAppVolume() *pluginsdk.Resource {
 			"snapshot_directory_visible": {
 				Type:     pluginsdk.TypeBool,
 				Optional: true,
-				Computed: true,
+				Default:  true,
```

Comment:
I'm worried that this could be a breaking change if this hasn't been returned as true by the api for older netapp volumes. Are you able to confirm that this won't break existing resources?

Reply:
I confirmed that this returns true when not passed.